doc/classDiagram.mer

```diff
@@ -0,0 +1,41 @@
+---
+title: "Class relationships and selected members"
```

Comment:
Technically not "selected members" as I think you've included all the member methods and attributes.